src/pages/store/Store.tsx

```diff
@@ -9,65 +9,68 @@ import kakaoServie from '@services/kakaoService'
 import { StoreWrapper, StoreMapWrapper } from './Store.styles'
 import { fetchAllReviews } from '@stores/review/reivewSlice'
 import { useSelector } from 'react-redux'
+import LoadingWithLogo from '@styles/LoadingWithLogo'
 
 const Store = () => {
   const dispatch = useAppDispatch()
-
+  const { storeId } = useParams<string>()
   const selectedStore = useSelector(
     (state: RootState) => state.conv.selectedStore
   )
-  const { storeId } = useParams<string>()
+  const loading = useSelector((state: RootState) => state.review.loading)
   const mapRef = useRef<HTMLDivElement | null>(null)
-  const [myPosition, setMyPosition] = useState<{ lat: number; lng: number }>({
-    lat: 37.54699,
-    lng: 127.09598,
+  const [defaultCoord, setDefaultCoord] = useState({
+    x: '127.09598',
+    y: '37.54699',
   })
 
-  useEffect(() => {
-    if (mapRef.current) mapRef.current.innerHTML = ''
-    const mapContainer = mapRef.current as HTMLDivElement
-    if (selectedStore) {
-      if (!selectedStore.y) {
-        const mapOption = {
-          center: new kakao.maps.LatLng(myPosition.lat, myPosition.lng), // 지도의 중심좌표
-          level: 3, // 지도의 확대 레벨
-        }
-        const map = new kakao.maps.Map(mapContainer, mapOption)
-        // 임의의 편의점 위치에 마커 생성
-        kakaoServie.displayMyLocation(
-          map,
-          new kakao.maps.LatLng(myPosition.lat, myPosition.lng)
-        )
-      } else {
-        const mapOption = {
-          center: new kakao.maps.LatLng(
-            Number(selectedStore.y),
-            Number(selectedStore.x)
-          ), // 지도의 중심좌표
-          level: 3, // 지도의 확대 레벨
-        }
-        const map = new kakao.maps.Map(mapContainer, mapOption)
-        // 임의의 편의점 위치에 마커 생성
-        kakaoServie.displayMyLocation(
-          map,
-          new kakao.maps.LatLng(
-            Number(selectedStore.y),
-            Number(selectedStore.x)
-          )
-        )
-      }
-    }
-  }, [selectedStore, myPosition.lat, myPosition.lng])
-
   useEffect(() => {
     if (storeId) {
       dispatch(fetchStoreInfo(storeId))
       dispatch(fetchAllReviews(storeId))
     }
   }, [storeId, dispatch])
 
+  useEffect(() => {
+    if (mapRef.current) mapRef.current.innerHTML = ''
+    const mapContainer = mapRef.current as HTMLDivElement
+    if (selectedStore) {
+      console.log(selectedStore)
+      const [storeBrand] = selectedStore.place_name.split(' ')

```

Comment:
(p4) split 두번째 파라메터가 limit 인데, 성능 개선하는 여지가 있네요
```suggestion
      const [storeBrand] = selectedStore.place_name.split(' ', 1)
```